raiden_installer/transactions.py

```diff
@@ -23,19 +23,19 @@ def _make_deposit_proxy(w3: Web3, token: Erc20Token):
 
     service_token_address = to_checksum_address(proxy.functions.token().call())
 
-    if service_token_address != to_checksum_address(token.address):
+    if service_token_address != token.address:
```

Comment:
Why would you compare it as a checksummed string and not as binary address? In general string representation should only be used for readability purposes. (Might be messed up in the wizard, but worth fixing it)

Reply:
Well, web3.py returns all addresses as checksummed strings (the call to `to_checksum_address` in line 24 is actually unnecessary). Would you change all addresses to be stored as bytes or just change the comparisons?

Reply:
I'd rather turn all internal address storing into bytes consistently and use string representation only for logging purposes.

Reply:
Ok I'll do that

Reply:
thanks